server/src/main/java/org/elasticsearch/search/SearchHit.java

```diff
@@ -167,6 +171,25 @@ public long getVersion() {
         return this.version;
     }
 
+
+    public void setSeqNo(long seqNo) {
+        this.seqNo = seqNo;
+    }
+
+    public void setPrimaryTerm(long primaryTerm) {
+        this.primaryTerm = primaryTerm;
+    }
+
```

Comment:
Can you add these fields in SearchHitTests#createTestItem to test all types of (de)serialization ?